tests/features/warmshowers/header_navigation.feature

```diff
@@ -0,0 +1,71 @@
+#language:en
+Feature:I can use the header to navigate the site
+	In order to effectively use the site
+	As a new or registered user 
+	I can navigate the site using the header
+
+Scenario: I can use the logo to return to the Warmshowers Homepage
+	Given I am on any Warmshowers page
+	When I click on the Warmshowers Towel logo
+	Then I see the Warmshowers homepage
+
+#There is currently one exception to this- when "Expand Map" is clicked, the Home, My Profile, FAQ, and Forums links disappear from the header.  The size of the header does not change, so not sure why these links can't/shouldn't stay. 

```

Comment:
I like. Not a bad place to put the little issues you find with these steps that you think will need to be addressed in the upgrade.